docs/PROCESS

```diff
@@ -0,0 +1,36 @@
+# Process
+
+## Introduction
+
+This file documents tricks and customizations made to the default docusaurus
+configuration for future editors and even open-source contributors.
+
+## Code blocks
+
+At the time of the Q3 documentation sprint, docusaurus had not well documented
+the code block formatting/prettifying, nor provided options for prefixing each
+line. For that reason, we did a little research and found that putting `json`
```

Comment:
Isn't that part of the base specs of Markdown?

Reply:
yes, but there are like 15 "flavors" of markdown and only some of them do this...